src/ray/core_worker/transport/raylet_transport.cc

```diff
@@ -48,7 +48,8 @@ void CoreWorkerRayletTaskReceiver::HandleAssignTask(
   }
 
   std::vector<std::shared_ptr<RayObject>> results;
-  auto status = task_handler_(task_spec, resource_ids, &results);
+  ReferenceCounter::ReferenceTableProto borrower_refs;
```

Comment:
maybe leave a comment that reference counting doesn't work for raylet transport? this will be deprecated soon but still may be confusing